netlify.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+command = "pnpm build"
```

Comment:
does Netlify absolutely need this? is there any way to get it to automatically figure out that it uses `pnpm` without configuration? (eg. would be great if it would just figure it out from the `pnpm-lock.yaml` file)

Reply:
this may be relevant for all of the projects we want to deploy on Netlify during the course - so you may need to also update the instructions in lectures / the Command Line cheatsheets

Reply:
I don't think Netlify can figure out the build commands from the `pnpm-lock.yaml` file, at least, I couldn't find anything. I was able to find out that In addition to using the Netlify UI to configure build settings, deploy settings, and environment variables, we can also configure these settings in a `netlify.toml` file, and with this file, I can overwrite the Netlify UI config. That's why I added this file, so we don't have to switch the build command on Netlify. [Link](https://docs.netlify.com/configure-builds/file-based-configuration/).

Reply:
I think that since we use `yarn build` as the build command on Netlify, it will always be used unless we create a `netlify.toml` file to change the build command

Reply:
`npm run build` is the default, right? since it's just running a script, I don't think it's different between `pnpm` and `npm` - we can just leave this right?